package.json

```diff
@@ -11,7 +11,8 @@
     "prepublishOnly": "npm run build",
     "bundle": "cd tools/bundler && npm i && npm run bundle",
     "startNewVersion": "newVersion=$npm_config_new_version node scripts/add-new-version.js",
-    "lint": "echo 'No linter integrated yet'",
+    "lint": "npx eslint . --ext .js",
```

Comment:
Looking at https://github.com/asyncapi/spec-json-schemas/issues/436, was the intention there to add JavaScript linting as well? After reading the issue I'm still not sure.

Reply:
No, it wasn't. And I think it will be better to remove it from this PR and rather add it later on. The only requirement I would say we should ask for in any PR is to, at least, a basic linting by eslint on each code pushed. Even though running it locally.

Reply:
so should I remove the JavaScript linting from this PR?
open a new issue and a follow-up PR for that issue adding the JS linting?

Reply:
> so should I remove the JavaScript linting from this PR? open a new issue and a follow-up PR for that issue adding the JS linting?

Sounds like a plan

Reply:
> Sounds like a plan

Added it to my To-do list. Let's do this next :rocket:

Reply:
marking as resolved because it will converted into a new issue in the future...